### ISSUE TITLE
webpack5.0.0 build failed 

### ISSUE THREAD

**Issue by chenjiajing23** [webpack/webpack#11638]
# Bug report

**What is the current behavior?**
this is a project example： https://github.com/chenjiajing23/react-template-pc/tree/feature/webpack-5
webpack-cli v4.0.0
```ts
[webpack-cli] 
Error: Universal Chunk Loading is not implemented yet
```

![image](https://user-images.githubusercontent.com/41351722/95680363-b1acd680-0c0b-11eb-948f-1731be1f8c1b.png)

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  5.0.0
Node.js version: 14.13.1
Operating System: mac os 10.15.7
Additional tools:vscode

**Comment by evilebottnawi** [comment]
https://github.com/webpack/webpack/releases/tag/v5.0.0-rc.1, please remove node from browserslist https://github.com/browserslist/browserslist#browsers, you don’t need universal chunk, or change the target option

**Comment by puilp0502** [comment]
@evilebottnawi  is correct. I have written a more in-depth explanation in #11660.

TL;DR: Set `output.chunkLoading` and `output.wasmLoading` to false.
